tests/snakemake/test_snakemake_anim_workflow.py

```diff
@@ -64,7 +65,7 @@ def config_filter_args(
     return {
         "outdir": anim_nucmer_targets_filter_outdir,
         "indir": str(input_genomes_small),
-        "cores": 8,
+        "cores": min(8, available_cores),
```

Comment:
Given the number of times `min(8, available_cores)` comes up, along with some other configurable settings that apply across multiple tests, can we assign this in something like `tools.py`/`config.py` under the `tests` directory, and import it instead?

Reply:
Or as yet another pytest fixture?

Rather than in the tests, I'd lean to adding a helper function within pyani-plus itself. Something like https://github.com/peterjc/thapbi-pict/blob/v1.0.13/thapbi_pict/__main__.py#L42 although probably under a new ``tools.py`` or ``utils.py`` module?

Reply:
I think you're right - a package-level module like `utils.py` would be a more sensible home. We could use that as a default for the CLI.

Reply:
For now I replaced the ``available_cores`` fixture with a ``snakemake_cores`` fixture.